iceoryx_utils/test/moduletests/test_cxx_convert.cpp

```diff
@@ -44,6 +44,18 @@ class convert_test : public Test
 };
 
 
+TEST_F(convert_test, toString_uint8_t)
+{
+    uint8_t data = 123U;
+    EXPECT_THAT(iox::cxx::convert::toString(data), Eq("123"));
+}
+
+TEST_F(convert_test, toString_int8_t)
+{
+    int8_t data = 123;
```

Comment:
Could you please vary the input data in every test a bit. Sometimes it can happen that the code is failing but since we provided the same input it is possible that some remainings in the memory will come to the expected outcome. If we vary the input data this problem becomes very unlikely.

As a suggestion, use `33331` (it is a prime - I like primes)

Reply:
done